perllib/FixMyStreet/Cobrand/Kingston.pm

```diff
@@ -91,4 +91,21 @@ Kingston starts collections at 6:30am, and lets you cancel up until then.
 sub bulky_collection_time { { hours => 6, minutes => 30 } }
 sub bulky_cancellation_cutoff_time { { hours => 6, minutes => 30 } }
 
+=head2 suppress_report_sent_email
+
+For Bulky Waste reports, we want to send the email after payment has been confirmed, so we
+suppress the email here.
+
+=cut
+
+sub suppress_report_sent_email {
```

Comment:
This function isn't being used anywhere, so the email is still being sent at the normal time.